From 40d12dbf97d4d165c094d7e4b1afa8ecc538c195 Mon Sep 17 00:00:00 2001
From: Longlong Xia <xialonglong1@huawei.com>
Date: Thu, 3 Nov 2022 12:46:27 +0000
Subject: [PATCH 263/595] [Huawei] lz4: add a config for this feature

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187930

----------------------------------------

An option CONFIG_CRYPTO_LZ4 and CONFIG_LZ4_COMPRESS is added to control the
enabling of the LZ4 algorithm with 4 KB fixed-sized compression for high
performance.

Signed-off-by: Xia Longlong <xialonglong1@huawei.com>
---
 arch/x86/configs/openeuler_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 67478ceeea0e..ef7dd84c0d6c 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -7852,7 +7852,7 @@ CONFIG_CRYPTO_TWOFISH_AVX_X86_64=m
 CONFIG_CRYPTO_DEFLATE=y
 CONFIG_CRYPTO_LZO=y
 # CONFIG_CRYPTO_842 is not set
-# CONFIG_CRYPTO_LZ4 is not set
+ CONFIG_CRYPTO_LZ4=m
 # CONFIG_CRYPTO_LZ4HC is not set
 CONFIG_CRYPTO_ZSTD=y
 
@@ -7988,6 +7988,7 @@ CONFIG_ZLIB_INFLATE=y
 CONFIG_ZLIB_DEFLATE=y
 CONFIG_LZO_COMPRESS=y
 CONFIG_LZO_DECOMPRESS=y
+CONFIG_LZ4_COMPRESS=m
 CONFIG_LZ4_DECOMPRESS=y
 CONFIG_ZSTD_COMPRESS=y
 CONFIG_ZSTD_DECOMPRESS=y
-- 
2.17.1

